src/test/librbd/object_map/test_mock_SnapshotRemoveRequest.cc

```diff
@@ -268,18 +268,27 @@ TEST_F(TestMockObjectMapSnapshotRemoveRequest, ScrubCleanObjects) {
 
   librbd::ImageCtx *ictx;
   ASSERT_EQ(0, open_image(m_image_name, &ictx));
+  librbd::NoOpProgressContext prog_ctx;
+  uint64_t size = 4294967296; // 4GB = 1024 * 4MB
```

Comment:
Nit: I believe in the test cases the default object size is 2MB (but you can confirm via ```ictx->get_object_size()```)

Reply:
yeah, the default object size is 2MB.
the size is computed by the original test code (objectmap.resize(1024)).